src/test/java/myPackage/Driver.java

```diff
@@ -3,13 +3,25 @@
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.chrome.ChromeDriver;
 
-public class Driver extends Elements{
+public class Driver extends Elements {
+    public ChromeDriver driver;
 
     public ChromeDriver driver() {
         // opens chrome browser
-        System.setProperty("webdriver.chrome.driver", "drivers/chromedriver");
+        //System.setProperty("webdriver.chrome.driver", "drivers/chromedriver");
+        String os = System.getProperty("os.name");
```

Comment:
make the driver() to pass the type of driver we want. 

 public ChromeDriver driver(String Type) { 

}